mitchellcomerlab1/lab1_hard_project/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.24)
+project(hello_world LANGUAGES CXX)
+
+add_library(hellolib STATIC src/hello_world.cpp include/hello_world.h)

```

Comment:
you do not need to include your header file as part of your library.

the header is already available to your library with your target include directories command